webservice/app.py

```diff
@@ -83,10 +99,14 @@ def all():
     urls = []
     for root, dirs, files in os.walk(app.config['UPLOAD_FOLDER']):
         for filename in files:
-            if allowed_file(filename):
-                name = get_name(filename)
+            extension = get_extension(filename)
+            name = get_name(filename)
+            if extension in ALLOWED_EXTENSIONS['pic'] or extension.lower() == 'jpg':
```

Comment:
is there a reason you don't write

extension.lower() in ALLOWED_EXTENSIONS['pic']

Reply:
solved